File: Per1/Latihan B/Py/tim.py
class tim():
    __nama = ""
    __negara = ""
    __tahun = 0
    __pelatih = ""
    __pemain = ""
    
    def __init__(self, nama = "", negara = "", tahun = 0, pelatih = "", pemain = ""):
        self.__nama = nama
        self.__negara = negara
        self.__tahun = tahun
        self.__pelatih = pelatih
        self.__pemain = pemain
    def setNama(self, nama):
        self.__nama = nama
    def getNama(self):
        return self.__nama
    def setNegara(self, negara):
        self.__negara = negara
    def getNegara(self):
        return self.__negara
    def setTahun(self, tahun):
        self.__tahun = tahun
    def getTahun(self):
        return self.__tahun
    def setPelatih(self, pelatih):
        self.__pelatih = pelatih
    def getPelatih(self):
        return self.__pelatih
    def setPemain(self, pemain):
        self.__pemain = pemain
    def getPemain(self):
        return self.__pemain

File: Per1/Latihan B/java/tim.java
public class tim {
    private String nama;
    private String negara;
    private int tahun;
    private String pelatih;
    private String pemain;
    public tim(String nama, String negara, int tahun, String pelatih, String pemain){
        this.nama = nama;
        this.negara = negara;
        this.tahun = tahun;
        this.pelatih = pelatih;
        this.pemain = pemain;
    }
    tim(){
    }
    public void setNama(String nama){
        this.nama = nama;
    }
    public String getNama(){
        return this.nama;
    }
    public void setNegara(String negara){
        this.negara = negara;
    }
    public String getNegara(){
        return this.negara;
    }
    public void setTahun(int tahun){
        this.tahun = tahun;
    }
    public int getTahun(){
        return this.tahun;
    }
    public void setPelatih(String pelatih){
        this.pelatih = pelatih;
    }
    public String getPelatih(){
        return this.pelatih;
    }
    public void setPemain(String pemain){
        this.pemain = pemain;
    }
    public String getPemain(){
        return this.pemain;
    }
}
